.travis.yml

```diff
@@ -53,8 +53,9 @@ matrix:
     # OSX {
     - os: osx
       env: PROJECT_DIR=examples/Libevent TOOLCHAIN=libcxx
-    - os: osx
-      env: PROJECT_DIR=examples/Libevent TOOLCHAIN=osx-10-11
+    # FIXME: https://travis-ci.org/isaachier/hunter/builds/357077343
```

Comment:
It's just an old set of toolchains, they have to be updated first. I've pushed changes:
* https://travis-ci.org/ingenue/hunter/builds/357391928

Reply:
I was afraid of that. Thanks. I'll wait for that to build.

Reply:
Looks good.